common/src/main/java/tachyon/Constants.java

```diff
@@ -280,4 +280,6 @@
   // Authorization
   public static final String TFS_PERMISSIONS_UMASK_KEY = "tachyon.tfs.permission.umask";
   public static final int DEFAULT_TFS_PERMISSIONS_UMASK = 0022;
+  public static final short DEFAULT_TFS_FULL_PERMISSION = (short)0777;
```

Comment:
I believe there should be a space after `(short)` according to Google java style guide 
http://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace